unlock-app/src/pages/api/auth/[...nextauth].ts

```diff
@@ -1,21 +1,85 @@
-import NextAuth from 'next-auth'
 import GoogleProvider from 'next-auth/providers/google'
+import EmailProvider from 'next-auth/providers/email'
```

Comment:
As mentioned, we do NOT want to have server logic in `unlock-app`. Please reconsider.